src/installer/managed/Microsoft.NET.HostModel/AppHost/HostWriter.cs

```diff
@@ -143,10 +145,8 @@ void UpdateResources()
                     }
                 }
 
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
-                {
-                    CodeSign($"-s - {appHostDestinationFilePath}");
-                }
+                if (enableMacOSCodeSign)
+                    CodeSign(appHostDestinationFilePath);
```

Comment:
I would still keep the platform guard here.  It's mostly about the fact that the implementation simply won't work anywhere else - ideally the API to sign would not even exist on non-mac (but that's just too much work and not worth the trouble).